### ISSUE TITLE
Typescript definitions

### ISSUE THREAD

**Issue by cliedeman** [jochen-schweizer/express-prom-bundle#32]
Would you be open to a PR thats adds typescript type definitions?

**Comment by disjunction** [comment]
don't have much experience with TS, but as long as it's not breaking anything, why not!

**Comment by disjunction** [comment]
@cliedeman thanks for this and also for whitelisting the files in `package.json`!

FYI it's now published as v5.1.1

**Comment by ajilo297** [comment]
@cliedeman Thanks for the type definition. But I have an error with
> 'normalizePath', which lacks return-type annotation, implicitly has an 'any' return type.
at this line.
```typescript
export function normalizePath(
  req: express.Request,
  opts?: { normalizePath?: NormalizePathRegexs }
);
```
Is it possible to change this to 
```typescript
export function normalizePath(
  req: express.Request,
  opts?: { normalizePath?: NormalizePathRegexs }
): any;
```
without breaking anything?

**Comment by cliedeman** [comment]
@ajilo297 My bad I missed the return value.

I created #34 to fix this.

Here is the [code](https://gist.github.com/cliedeman/e11278d4c541d5dd6e7a07bfd66c8e5f) I use to test the typedefs

Taking quick look we should be able to return string there.
https://github.com/disjunction/url-value-parser/blob/master/src/UrlValueParser.js#L35

Ciaran

**Comment by ajilo297** [comment]
@cliedeman Awesome. Thanks

**Comment by disjunction** [comment]
published as v5.1.2

**Comment by cliedeman** [comment]
Thanks and sorry for the churn
